### PR TITLE
Add Harrisburg file

### PR DESCRIPTION
**Pull request by aharden**
This is the beginning of the Harrisburg file. 🎸 

Please provide feedback on:

- Ever been to Harrisburg?
- Any ideas for other attractions?
- Kids activities?
- Good 🍺 breweries?

### CODE REVIEW

File: aharden-harrisburg.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### Restaurants
+
+### Local Attractions
+
+- National Civil War Museum
+- Pennsylvania State Capitol
+- Pennsylvania State Museum
+
+### Remote Attractions
```
Comment (readiu) [inline aharden-harrisburg.md]: Create town :+1:

Comment (readiu) [inline aharden-harrisburg.md]: Great town :+1:

**Comment by sdcalmes** [comment]
:ship:

**Comment by beardofedu** [comment]
:shipit: never been to Harrisburg but I think this is a great PR

**Comment by githubteacher** [comment]
I agree with @sdcalmes and @beardofedu  -- this looks ready to 🚢

**Pull request merged**